### DISCUSSION TITLE
BUG in geocore ce /main catagories only 3 in listing process

### DISCUSSION THREAD

**Discussion by Genovit** [geodesicsolutions-community/geocore-community#159] [Q&A]
Hi i set up my classifieds an auction enabled ,i started the cart listing process fpr classifieds post only 3 out of 7 mian catagoreis show up in the main box pick,but when i start same process for auction listiing they all show up! can some one point me to how to fix this been at it for 3 hrous lol thanks

**Comment by Genovit** [comment]
hers two small screen shots
![AUCTION2](https://user-images.githubusercontent.com/16966910/160222715-80270b97-f339-4fa4-93d7-163bdf55694e.png)
![CLASS1](https://user-images.githubusercontent.com/16966910/160222717-1fda18da-7181-4f3d-9461-90b61454b604.png)

**Comment by vicos59** [comment]
![Capture](https://user-images.githubusercontent.com/6634041/160234523-18b3b84e-76e9-4ea6-8987-3a34082be385.JPG)

Did you edit the category to confirm that both Auction and Classifieds are selected, as shown above?

**Comment by Genovit** [comment]
yes i made sure still not showing up,

**Comment by vicos59** [comment]
I just checked my test site and the same categories are showing up for both classifieds and auctions. 

I have never used auctions for my real site, so I can't offer you any more insight. Try retracing your steps to see what you might have done along the way to disable those. You probably need to make sure the parent category, if any, is also enabled for auctions.

**Reply by Genovit** [reply]
fond the problem in classifieds price plan theres a row of ban check boxes they were excluded from the classified listing process ,sorry to wast your time ,
![ban-check](https://user-images.githubusercontent.com/16966910/160238067-2b9a0091-495f-42be-9968-ce3ee65f8743.png)

**Comment by vicos59** [comment]
Hi @Genovit,

Just wanted to check with you to see how everything is going with the software. Is everything working out? Are you using a payment gateway?

**Reply by Genovit** [reply]
The only thing im having problems with is trying to creat new user group for banner advertizers,
iv installed the enterprize price plan,cannot seem to make new subscription price plan outside the classified default one or make new user group
its like witch comes first? the new price plan or user group? im stuck on new user group page says to ad experation, in red witch i did but it will not let me select the subscription part as show in pitcher,wish there was a youtube video on this lol as far as gateways iv seleted stripe seems to work ok,have not done a test run yet!

**Comment by Genovit** [comment]
![usergroup](https://user-images.githubusercontent.com/16966910/161148336-625e6b88-da50-41cf-97db-f505676832bd.png)

**Reply by Genovit** [reply]
iv made a subscription  based plan but when i try to select the subscription buttin it will not let me select in red iv done what it said ,plan is there with cost an expiration time ,dont get it.lol

**Reply by vicos59** [reply]
Does the 'Silver Auctions Plan' have a specific time period assigned to it, i.e. it renews every  30 days?

I assume 'Fee Based' is a one-time charge and 'Subscription' plans renew every period (xx days). Thus the price plans attached to a subscription-based group must have a specific time frame assigned that will renew.

**Reply by Genovit** [reply]
IL check it out,i wanted to make a user group for some kind of subscription banner advertizing.

**Reply by Genovit** [reply]
![s-auct-1](https://user-images.githubusercontent.com/16966910/161431044-468be162-63bb-4467-8b45-883e5bae5143.png)
![s-auct-2](https://user-images.githubusercontent.com/16966910/161431045-0e15e3de-309d-43d9-9d91-92aa478f755d.png)

**Reply by Genovit** [reply]
this is what i have now
